src/linker/configuration.py

```diff
@@ -21,44 +21,34 @@ def __init__(
             self.computing_environment_path = None
         else:
             self.computing_environment_path = Path(computing_environment)
+        self.input_data_path = Path(input_data)
```

Comment:
Is there a reason this is not converted to a Path in the cli like we have done in other places?

Reply:
Implemented in followup PR